content/en/platform/corda/5.0-beta/developing/getting-started/installing-corda-cli.md

```diff
@@ -26,21 +26,21 @@ Java     | Azul JDK 11
 
 To obtain the Corda CLI installer from the release pack on Customer Hub:
 1. Download `platform-jars-Gecko.tar.gz` from the [R3 Customer Hub](https://r3.force.com/) or from the [R3 S3 repository](https://download.corda.net/packages/corda-cli-downloader/5.0.0.0-Gecko1.0/corda-cli-downloader-5.0.0.0-Gecko1.0.zip).
-2. Extract `corda-cli-installer-5.0.0.0-Gecko1.0.zip` from `net\corda\cli\deployment\corda-cli-installer\5.0.0.0-Gecko1.0`.
+2. Extract `corda-cli-downloader-5.0.0.0-Gecko1.0.zip` from `net\corda\cli\deployment\corda-cli-installer\5.0.0.0-Gecko1.0`.
```

Comment:
@dimosr is the path you're extracting from correct? `net\corda\cli\deployment\corda-cli-installer\5.0.0.0-Gecko1.0`

Reply:
Checked on the wiki and it is a correct path https://r3-cev.atlassian.net/wiki/spaces/CB/pages/4420042753/C5+Beta+Program+-+Beta+3.0+Release+-+HC02

Reply:
I followed the instructions on the public docs here: https://docs.r3.com/en/platform/corda/5.0-beta/developing/getting-started/installing-corda-cli.html

Clicking on the "R3 S3 repository" link does not download a file called `platform-jars-Gecko.tar.gz`, as explained in the instructions. It downloads a file called `corda-cli-downloader-5.0.0.0-Gecko1.0.zip`.

I can see two issues here:
* Step 2 is wrong as there's no need to extract anything. The zip file is already the one you download.
* Even if you take that file and use it with the commands below, they won't work since they expect a file called `corda-cli-installer-5.0.0.0-Gecko.zip` and what you get is `corda-cli-downloader-5.0.0.0-Gecko1.0.zip`.